libmscore/layout.cpp

```diff
@@ -4579,14 +4619,18 @@ void Score::doLayoutRange(const Fraction& st, const Fraction& et)
 void LayoutContext::layout()
       {
       MeasureBase* lmb;
+      dominantScore = static_cast<MasterScore*>(score); // set the first score (masterscore = movement) as the dominant
```

Comment:
Can't you use <code>toScore(score)</code> instead of this cast?

Reply:
`toScore` seems to return a `Score*` not a `MasterScore*`

Reply:
And there is no <code>toMasterScore()</code> (unless you add it in <code>libmscore/scoreElement.h</code> :wink:). In general we try to prevent these kind of casts.